### PR TITLE
Fix: Add "OK" and "Cancel" to translation #214 [wip]

### PR DESCRIPTION
**Pull request by kas70**
Datepicker and some other fragments still don't use translatable button strings.

### CODE REVIEW

**Comment by kas70** [comment]
#214

File: src/de/grobox/liberario/fragments/RecentTripsFragment.java [hunk 1]
```diff
@@ -301,26 +301,26 @@ public boolean onActionItemClicked(final ActionMode mode, MenuItem item) {
 				case R.id.action_trip_delete:
 					new AlertDialog.Builder(getActivity())
 					.setMessage(getActivity().getResources().getString(R.string.clear_recent_trips, getListView().getCheckedItemCount()))
-					.setPositiveButton(android.R.string.yes, new DialogInterface.OnClickListener() {
```
Comment (grote) [inline src/de/grobox/liberario/fragments/RecentTripsFragment.java]: Yes/No should not be Ok/Chancel, but stay yes/no please.

**Comment by kas70** [comment]
@grote Should be ready for a review.

File: res/values-de/strings.xml [hunk 2]
```diff
@@ -284,4 +284,6 @@ es welche gibt) anzeigen lassen.</string>
   <string name="catalan">Katalanisch</string>
   <string name="polish">Polnisch</string>
   <string name="tamil">Tamil</string>
+  <string name="cancel">Abbrechen</string>
+  <string name="ok">Ok</string>
```
Comment (grote) [inline res/values-de/strings.xml]: Thanks, but there's no need adding them here, as this is overwritten when pulled from Transifex next time.

File: res/xml/preferences.xml [hunk 3]
```diff
@@ -64,6 +67,7 @@
             android:entryValues="@array/pref_language_values"
             android:defaultValue="@string/pref_language_value_default"
             android:summary="%s"
+            android:negativeButtonText="@string/cancel"
```
Comment (grote) [inline res/xml/preferences.xml]: Nice, didn't know that was possible.

**Comment by grote** [comment]
Looks good, need to test later  before merging. Thanks! :)

Do you want to squash your commits into one and maybe add two new strings for yes/no as well?

**Comment by kas70** [comment]
Done.

**Comment by grote** [comment]
did you forget to replace `android.R.string.yes/no` by `R.string.yes/no` ?

**Comment by kas70** [comment]
Yep, sorry.

**Comment by grote** [comment]
Thanks a lot, merged! :)

**Pull request closed without merging**